ScottWu.md

```diff
@@ -16,21 +16,21 @@
 
 - General features
 	- [feedback?] Passing data between input field and pickers
-	- Keyboard Controls
-		- Jessica will handle keyboard events, UI only receives changes made
+	- [WIP] Keyboard Controls
+		- Input box handles keyboard events most of the time, until user triggers shifting "visual focus" to picker
```

Comment:
I need to prepare my picker to receive a "visual focus" event from input box. From that point on, input box will stop changing value based on keyboard event, and will instead pass me the events and let the picker respond to them.